examples/gltf/src/main.rs

```diff
@@ -10,6 +10,12 @@ fn load_gltf(
 
     let vertex_positions: Vec<_> = reader.read_positions().unwrap().map(glam::Vec3::from).collect();
     let vertex_normals: Vec<_> = reader.read_normals().unwrap().map(glam::Vec3::from).collect();
+    let vertex_tangents: Vec<_> = reader
+        .read_tangents()
+        .unwrap()
+        .map(glam::Vec4::from)
+        .map(From::from)
```

Comment:
I just realized I've completely forgotten about tangent handedness. In most cases `w` will just be 1, but for example when you use mirrored meshes where UV's go in different directions this can be -1 and means the bitangent should be flipped.

Reply:
I'm not too experienced with normal mapping, but I'm perfectly fine if you want to hold off on dealing with this in this pr. Just file a follow up issue to track what would have to be fixed.